src/realm/table.cpp

```diff
@@ -292,6 +292,20 @@ void Table::insert_column_link(size_t col_ndx, DataType type, StringData name, T
 }
 
 
+size_t Table::get_backlink_count(size_t row_ndx) const noexcept
+{
+    size_t first_backlink_column = m_spec->first_backlink_column_index();
+    size_t last_backlink_column = first_backlink_column + m_spec->backlink_column_count();
```

Comment:
This shouldn't be named "last_backlink_column" since it points one beyond the last (I had to double check that there wasn't a bug in the for-loop that follows). Maybe "backlink_columns_[begin|end]" instead, like in STL?